spec/browserstack_helper.rb

```diff
@@ -1,40 +1,40 @@
-# -*- coding: utf-8 -*-
```

Comment:
It didn't even have to look quite that weird(1): this would have been adequate:

    # encoding: utf-8

However, as of Ruby 2.0, the default is UTF-8, so this is completely redundant.

(1) Emacs-y